mindmeld/query_factory.py

```diff
@@ -31,10 +29,10 @@ class QueryFactory:
         tokenizer (Tokenizer): the object responsible for normalizing and tokenizing processed
             text
     """
-    def __init__(self, tokenizer, preprocessor=None):
+    def __init__(self, tokenizer, preprocessor=None, stemmer=None):
         self.tokenizer = tokenizer
         self.preprocessor = preprocessor
-        self.stemmer = nltk.stem.PorterStemmer()
+        self.stemmer = stemmer or EnglishNLTKStemmer()
```

Comment:
Can we move this default into `create_query_factory()` to be consistent with how the tokenizer default is set?

Reply:
good point, done.